src/JSInterop/Microsoft.JSInterop/src/JSInProcessRuntimeBase.cs

```diff
@@ -22,7 +22,9 @@ public TValue Invoke<TValue>(string identifier, params object[] args)
             var resultJson = InvokeJS(identifier, JsonSerializer.ToString(args, JsonSerializerOptionsProvider.Options));
             if (resultJson is null)
             {
+#pragma warning disable CS8653 // A default expression introduces a null value for a type parameter.
```

Comment:
I don't know how to use nullable + generics.

Reply:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/nullable-reference-types#generics

Emphasis:

> If a type parameter is unconstrained or has only nullable constraints, the situation is a little more complex: this means that the corresponding type argument could be either nullable or non-nullable. The safe thing to do in that situation is to treat the type parameter as both nullable and non-nullable, giving warnings when either is violated.

Reply:
I think you can probably `!default` it.